Packs/ipinfo/Integrations/ipinfo_v2/ipinfo_v2.py

```diff
@@ -60,7 +60,7 @@ def parse_results(ip: str, raw_result: Dict[str, Any], reliability: str) -> List
                                             entity_a_type=FeedIndicatorType.IP,
                                             entity_b=hostname,
                                             entity_b_type=FeedIndicatorType.Domain,
-                                            brand='IPinfo',
+                                            brand='ipinfo_v2',
```

Comment:
@DeanArbel should it stay `IPinfo`?

Reply:
this, and yml fields, id and name, should all be the same.

Reply:
Is changing this considered breaking BC?